### PR TITLE
fix error return but uncheck and fix bug

### PR DESCRIPTION
**Pull request by lemondy**
修改saveConfig 函数返回值，由nil修改为具体的错误类型
修复SaveConfigToFile遇到错误应该直接返回error，而不是继续执行下面的代码

### CODE REVIEW

File: cmd/gyee/configcmd.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ package main
 
 import (
 	"fmt"
-
+	"errors"
```
Comment (Chm) [inline cmd/gyee/configcmd.go]: system packages should be grouped in first block

File: cmd/gyee/configcmd.go [hunk 2]
```diff
@@ -57,16 +57,17 @@ Generate a a default config file.`,
...
 	fileName := ctx.Args().First()
 	if len(fileName) == 0 {
 		fmt.Println("please give a config file arg!!!")
-		return nil
+		return errors.New("please give a config file arg!!!")
 	}
 
 	node := makeNode(ctx)
-	config.SaveConfigToFile(fileName, node.Config())
+	err = config.SaveConfigToFile(fileName, node.Config())
```
Comment (Chm) [inline cmd/gyee/configcmd.go]: safe to just return here

Comment (lemondy) [reply cmd/gyee/configcmd.go]: pull a new request
你说的这里修改点我不是很明白，saveConfig函数修改了下，看是否ok

File: cmd/gyee/configcmd.go [hunk 3]
```diff
@@ -23,34 +23,32 @@ package main
...
 	"github.com/urfave/cli"
 	"github.com/yeeco/gyee/config"
 )
 
 var (
 	configCommand = cli.Command{
-		Name:     "config",
-		Usage:    "Manage config",
-		Category: "CONFIG COMMANDS",
-		Description: `
```
Comment (Chm) [inline cmd/gyee/configcmd.go]: leading newline here was shown in help text, no need to format

Comment (lemondy) [reply cmd/gyee/configcmd.go]: ok

**Comment by Chm** [comment]
I see u've got 7 commits with empty and merge along the way.
Could u plz squash them into a single commit?

**Comment by Chm** [comment]
master branch is protected by default, you may find a dedicated patch branch handy.

**Comment by lemondy** [comment]
OK

**Comment by lemondy** [comment]
you can close this pr

**Pull request closed without merging**